### ISSUE TITLE
sklearn.utils.fixes.logsumexp In pypi

### ISSUE THREAD

**Issue by IvanaXu** [scikit-learn-contrib/metric-learn#294]
In pypi 0.5.0, site-packages/metric_learn/nca.py can see that 
```
from scipy.optimize import minimize
from sklearn.metrics import pairwise_distances
from sklearn.exceptions import ConvergenceWarning, ChangedBehaviorWarning
from sklearn.utils.fixes import logsumexp
from sklearn.base import TransformerMixin
```
but in the source code 

https://github.com/scikit-learn-contrib/metric-learn/blob/43a60c9f4380448b9013efa4fb021796e7e1ff3c/metric_learn/nca.py#L9-L13

I see, it update this code to debug sklearn version ???
```
from sklearn.utils.fixes import logsumexp
```
how about to update the pypi???

**Comment by bellet** [comment]
Hi,

This problem was recently fixed in #290. The PyPi version will be updated when we make a new release (probably very soon). In the meantime, you can install the latest code from master (see README).
